clipper_admin/clipper_admin/deployers/caffe2.py

```diff
@@ -0,0 +1,159 @@
+from __future__ import print_function, with_statement, absolute_import
```

Comment:
This file shouldn't be called the Caffe2 deployer, because it doesn't let people deploy Caffe2 models. Let's rename this to `onnx.py` and we'll start to centralize all our ONNX-related model deployer functionality in here.